Enhanced documentation in time series section with feedback from support (#2967)
SoftwareAG/c8y-docs: content/standard-tenant/enhanced-time-series-support-bundle/general-configuration.md

The following APIs are not supported and do not have a replacement:

* `GET /measurement/measurements/{id}`
 

> Alternatively one can use the `GET /measurement/measurements` API to retrieve a list of measurements. By specifying the device(source) id and the exact point in time the measurement was sent by the device, the result set can be reduced significantly. Note, every measurement document returned, does still contain an identifier.  

Reviewer: ```suggestion
Alternatively, you can use the `GET /measurement/measurements` API to retrieve a list of measurements. By specifying the device (source) ID and the exact point in time when the measurement was sent by the device, the result set can be reduced significantly. Note that every returned measurement document still contains an identifier.  
```
